### PR TITLE
Docs generation fix

### PR DESCRIPTION
**Pull request by themuflon**
Added docs generation for enums and extended some text alignment comments.

### CODE REVIEW

File: loom/script/compiler/lsParser.cpp [hunk 1]
```diff
@@ -2572,6 +2572,9 @@ Statement *Parser::parseEnumStatement()
     cls->metaTags = curTags;
     curTags.clear();
 
+    cls->docString = currentMultilineComment;
+    currentMultilineComment = "";
```
Comment (SmilyOrg) [inline loom/script/compiler/lsParser.cpp]: What does setting it to `""` achieve here? Prevents the following APis getting the same comment? Is `currentMultilineComment` used anywhere else?

**Comment by SmilyOrg** [comment]
Just a question or two, if it works and build box is green, it's good to go though :)

File: loom/script/compiler/lsParser.cpp (hunk 1 above)
Comment (themuflon) [inline loom/script/compiler/lsParser.cpp]: This is a pattern that's used all over this code. It looks like it's a kind of a "used" flag. Could have been done cleaner with .clear() or something though, but then we'd probably want to make everything consistent.

Comment (SmilyOrg) [inline loom/script/compiler/lsParser.cpp]: Ah ok, sure.

**Pull request merged**